src/main/java/com/servicewizard/config/TransformerConfiguration.java

```diff
@@ -0,0 +1,83 @@
+package com.servicewizard.config;
+
+import org.hibernate.validator.constraints.NotEmpty;
+
+public class TransformerConfiguration {
+	/**
+	 * The type of transformer to use.
+	 */
+	public String getType() {
+		return type;
+	}
+
+	public void setType(String type) {
+		this.type = type;
+	}
+
+	public TransformerType getEnumType() {
```

Comment:
It feels wrong having Enum in the name. Could we name this getTransformerType instead?